src/app/views/common/download.ts

```diff
@@ -23,3 +23,9 @@ function trackDownload(filename: string) {
     filename
   });
 }
+
+export function trackGenerate() {
+  telemetry.trackEvent(eventTypes.BUTTON_CLICK_EVENT, {
+    ComponentName: componentNames.DOWNLOAD_POSTMAN_COLLECTION_BUTTON
```

Comment:
Should this be `componentNames.GENERATE_CLIENT_PREVIEW_BUTTON` ?

Reply:
thanks for the callout, fixed